functional_tests/testdata/expected_eks_values/expected_cluster_receiver.yaml

```diff
@@ -1150,6 +1174,42 @@ resourceMetrics:
                       value:
                         stringValue: k8scluster
                   timeUnixNano: "1000000"
+                - asInt: "1"
+                  attributes:
+                    - key: container.id
+                      value:
+                        stringValue: 38809b5e5212eddddeec2d4d387e2f27ff754e0c3f67cc9a9ffb89db72601480
```

Comment:
Is this id always the same?

Reply:
@pjanotti this id and others are ignored through https://github.com/signalfx/splunk-otel-collector-chart/pull/1276/files#diff-96b5efbb91350a0f6fd749aad1b820223a25daa5c5a0e6613c711815c3ae60f8R569-R586

Reply:
sorry wrong place but the ignores are in the metrics test, same mechanism. The test passes https://github.com/signalfx/splunk-otel-collector-chart/actions/runs/9306302191/job/25615214854?pr=1276#step:9:289